Docs refactor cli 2

blackbaud-johnly opened the pull request.

Blackbaud-BobbyEarl commented on src/app/learn/reference/cli-commands/index.html:
```diff
@@ -5,40 +5,19 @@
-    <li><p><stache-code>skyux e2e</stache-code> — Runs end-to-end tests that test your SPA running in a real browser, interacting with it as users would. The <stache-code>e2e</stache-code> command uses a tool called Protractor to look for files in the <stache-code>e2e</stache-code> folder with names that end in <stache-code>.e2e-spec.ts</stache-code>. End-to-end tests supplement the Karma unit te ...
-    <li><p><stache-code>skyux watch</stache-code> — Runs the <stache-code>test</stache-code> command and watches the file system for changes. The <stache-code>watch</stache-code> command monitors changes in much the same way as the <stache-code>serve</stache-code> command, but for testing, the <stache-code>watch</stache-code> command is always running, much like debug mode in Visual Studio. If yo ...
-    <li><p><stache-code>skyux lint</stache-code> — Runs the <a href="https://marketplace.visualstudio.com/items?itemName=eg2.tslint">TSLint extension</a> against the files in the <stache-code>src</stache-code> folder. The extension validates TypeScript code against the rules specified in the <stache-code>tslint.json</stache-code> file in the SKY UX 2 repo. Other commands such as <stache-code>skyu ...
+    <li><p><stache-code>skyux new</stache-code> — Initializes a new local SKY UX project. For more details, see <a routerLink="/learn/reference/cli-commands/new">the <stache-code>skyux new</stache-code> topic</a>.</p></li>
```
Hey @blackbaud-johnly... any thoughts on linking to the individual pages using a different layout or different verbiage?  It took me a bit to catch that "topic" was a link (ignoring that the links are in the sidebar too).

Blackbaud-SteveBrush replied:
I wonder if we even need those links? Or if this page should use the Action Buttons layout?

blackbaud-johnly replied:
I'd considered using the Action Buttons layout but was worried that I was going a little overboard with that pattern. Given that the links don't stand out though, I think it's probably the best option here. I'll work on that today. Thanks.

Blackbaud-SteveBrush commented on src/stache/includes/cli/service.html:
```diff
@@ -0,0 +1 @@
+<p><stache-code>--service</stache-code> or <stache-code>-s</stache-code> — Serves the current SPA locally after it builds into the <stache-code>dist/</stache-code> folder.</p>
```
I believe this should be `--serve`

Blackbaud-SteveBrush commented:
I think this looks pretty good. @Blackbaud-BobbyEarl?

Blackbaud-BobbyEarl commented:
I'm assuming the order of the commands is an approximation of the order you'd use them?  Any thoughts on that order versus just alphabetical?

blackbaud-johnly commented:
I mostly just stuck with the current order. I think we originally organized the commands based on combination of putting them in the order that you'd use them and an approximate estimation of how important they are. I can switch to alphabetical order if you prefer.

blackbaud-johnly commented:
@Blackbaud-BobbyEarl, I updated the CLI commands to follow alphabetical order, and I also fixed the redirects we talked about earlier. Can you please take a look when you get a chance? Thanks!

Blackbaud-BobbyEarl commented:
I definitely like the alphabetical listing better.  Thanks @blackbaud-johnly!  I could see in the not-so-distant future, having an "Extend" section there that I'm hoping @blackbaud-brandonhare is willing help contribute to.
